src/net.cpp

```diff
@@ -90,6 +90,9 @@ static constexpr std::chrono::seconds MAX_UPLOAD_TIMEFRAME{60 * 60 * 24};
 // A random time period (0 to 1 seconds) is added to feeler connections to prevent synchronization.
 static constexpr auto FEELER_SLEEP_WINDOW{1s};
 
+/** Frequency to attempt extra connections to reachable networks we're not connected to yet **/
+static constexpr auto EXTRA_NETWORK_PEER_INTERVAL = 5min;
```

Comment:
The commit message says every ~1min, but this means it will be ~5min via GetExponentialRand I think? This seems to be more in line with timings I have observed during testing, e.g. 

```log
2023-07-05T14:04:26Z [net] Making network specific connection to ipv6_addr:8333
2023-07-05T14:15:16Z [net] Making network specific connection to onion_addr.onion:8333
```